src/components/ModuleCard/ModuleCard.tsx

```diff
@@ -0,0 +1,138 @@
+import {Card, CardDeck, Row} from 'react-bootstrap';
+import React from 'react';
+import styles from './ModuleCard.module.scss';
+import TestImage from '../../assets/images/TestImage.jpg';
+
+export const ModuleCard = () => (
```

Comment:
A module card should just be be one card, hence it's singularity. This is an effort to reduce duplication, not move the duplication to another file.